Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -5841,6 +5841,9 @@ void CClientGame::ResetMapInfo()
             g_pNet->DeallocateNetBitStream(pBitStream);
         }
     }
+
+    // Reload world
+    RestreamWorld();
```

Comment:
```suggestion
    RestreamWorld();
```
The code comment is not needed because the name of the function is self-explanatory.